tools/tools.go

```diff
@@ -160,13 +160,13 @@ func StepmanStepInfo(collection, stepID, stepVersion string) (stepmanModels.Step
 
 // StepmanRawStepList ...
 func StepmanRawStepList(collection string) (string, error) {
-	args := []string{"--loglevel", logLevel(), "step-list", "--collection", collection, "--format", "raw"}
+	args := []string{"step-list", "--collection", collection, "--format", "raw"}
```

Comment:
The user can set the debug mode in two ways:
- env var
- command line parameter

The stepman process inherits the envs from the parent process so if the user used the debug env key then stepman can also print the debug logs.

But if the user used the command line parameters then the cli will print debug logs but stepman will not because we are not passing the loglevel anymore. 

I do not know yet how useful the stepman debug logs are so I am also happy to remove this. But if we want consistency then I would add the `--loglevel debug` parameter to stepman in the case if we are running in a debug mode.

Reply:
Updated my PR to restore the original behaviour around debug mode.
Now envan and stepman inherits Bitrise CLIs debug mode, this involved 2 main changes:
- improved how we parse debug mode flag (loggerParameters) + tests
- setting `LOGLEVEL=debug` env var

Specifying `--loglevel=debug` flag for the envman and stepman commands is not needed because the LOGLEVEL env var controls the log level flag for both [stepman](https://github.com/bitrise-io/stepman/blob/4d242ab03ef96430fc79b37201e519e1e5e5520d/cli/flags.go#L89) and [envman](https://github.com/bitrise-io/envman/blob/master/cli/flags.go#L80).

On the other side, specifying `--loglevel=debug` in itself is not enough, because this way we restore the behavior for wrapped commands of envman and stepman, but not for the envman and stepman plugin commands:
- Plugin command: `bitrise -debug stepman share audit`
- Wrapped command: `bitrise -debug share audit`